sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/webpack-dev-server/test

```diff
@@ -0,0 +1,17 @@
+# the server should properly start
+> fastOptJS::startWebpackDevServer
+# Give the server some time to launch
+# It's hacky, but this is just a test..
+$ sleep 3000
+> html index.html
+
+# the server should properly stop
+> fastOptJS::stopWebpackDevServer
+-> html index.html
+
+# the server should properly terminate on exit
+> fastOptJS::startWebpackDevServer
+$ sleep 3000
+> html index.html
+-> exit
```

Comment:
Because `scripted` thinks that this is an error (probably because SBT doesn't return back to prompt after this one). I found no other way to test the exit hook, though (`reload``isn't the same, since it runs unload hooks, and `exit` doesn't).